include/sym1.h

```diff
@@ -0,0 +1,154 @@
+#define DISP_S         0x6D   // 'S'
+#define DISP_t         0x46   // 't'
+#define DISP_U         0x3E   // 'U'
+#define DISP_u         0x1C   // 'u'
+#define DISP_V_1       0x64   // 'V'
+#define DISP_V_2       0x52   // 2nd half
+#define DISP_W_1       0x3C   // 'W'
+#define DISP_W_2       0x1E   // 2nd half
+#define DISP_Y         0x6E   // 'Y'
+#define DISP_Z         0x5B   // 'Z'
+#define DISP_SPACE     0x00   // ' '
+#define DISP_PERIOD    0x80   // '.'
+#define DISP_HYPHEN    0x40   // '-'
+#define DISP_APOSTR    0x20   // '''
+#define DISP_EQUAL     0x41   // '='
+#define DISP_3_BAR     0x49   // '='
+#define DISP_BOTTOM    0x08   // '_'
+#define DISP_TOP       0x01   // Top segment
+#define DISP_LEFT      0x30   // '|' Left side, both segments
+#define DISP_RIGHT     0x06   // '|' Right side, both segments
+#define DISP_DEGREE    0x63   // 'o' An 'o' character in the upper segments
+#define DISP_HAT       0x23   // 'n' An 'n' character in the upper segments
+#define DISP_FORK      0x62   // 'u' A  'u' character in the upper segments
+#define DISP_SLASH     0x51   // '/'
+#define DISP_BACKSLASH 0x34   // '\'
+#define DISP_TOP_RIGHT 0x02   // Top right segment
+#define DISP_TOP_LEFT  0x20   // Top left segment
+#define DISP_LOW_RIGHT 0x04   // Lower right segment
+#define DISP_LOW_LEFT  0x10   // Lower left segment
+
+
+
+/*****************************************************************************/
+/*                                   Code                                    */
+/*****************************************************************************/
+
+
+
+int __fastcall__ beep (void);                // Beep sound
+void __fastcall__ set_D0 (unsigned char);    // Set display digit 0
+unsigned char __fastcall__ get_D0 (void);    // Get value of display digit 0
+void __fastcall__ set_D1 (unsigned char);    // Set display digit 1
+unsigned char __fastcall__ get_D1 (void);    // Get value of display digit 1
+void __fastcall__ set_D2 (unsigned char);    // Set display digit 2
+unsigned char __fastcall__ get_D2 (void);    // Get value of display digit 2
+void __fastcall__ set_D3 (unsigned char);    // Set display digit 3
+unsigned char __fastcall__ get_D3 (void);    // Get value of display digit 3
+void __fastcall__ set_D4 (unsigned char);    // Set display digit 4
+unsigned char __fastcall__ get_D4 (void);    // Get value of display digit 4
+void __fastcall__ set_D5 (unsigned char);    // Set display digit 5
+unsigned char __fastcall__ get_D5 (void);    // Get value of display digit 5
+void __fastcall__ set_D6 (unsigned char);    // Set byte to the right of display (leading buffer)
+unsigned char __fastcall__ get_D6 (void);    // Get value of memory byte to the right of display
+void __fastcall__ fdisp (void);              // Flash display
+
+int __fastcall__ loadt (int);                // Read from tape (id)
+int __fastcall__ dumpt (int, int, int);      // Write to tape (id, start_addr, end_addr)
```

Comment:
Addresses for an output function are `const void*`, not `int`, see e.g. https://github.com/cc65/cc65/blob/master/include/unistd.h#L77

Reply:
Updated tapeio to expect const void* rather than int for start_addr and end_addr.